### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
+
+const averageAge = (people) => people.reduce(
+  (sum, person) => sum + person.died - person.born, 0)
+  / people.length;
+
```
Comment (naziamoff) [inline src/averageAges.js:21]: does `averageAge` name sounds good for a function?

**Review by naziamoff — APPROVED** [review]
great, super clear code. Just one thing:

**Review by naziamoff — APPROVED** [review]
great, super clear code. Just one thing:

File: src/averageAges.js (hunk 1 above)
Comment (Shevtsooov) [reply src/averageAges.js:21]: Got it, will correct, thanks!

**Pull request closed without merging**